### ISSUE TITLE
BUG: [Test Environments] Entry mocks do not include ZoomIn

### ISSUE THREAD

**Issue by angelo-hub** [software-mansion/react-native-reanimated#5674]
### Description

The entry mocks currently don't include the `ZoomIn` entry, creating issues with any snapshots that include them.

fix is pretty simple and i have in a patch file already of just adding ZoomIn as a mockable layout animation

```
+  'ZoomIn',
   'ZoomInRotate',
   'ZoomInLeft',
   'ZoomInRight',
```

### Steps to reproduce

1. create a component with a ZoomIn layout effect
2. try to create a jest snapshot of component
3. test will fail as it's undefined

### Snack or a link to a repository

N/A

### Reanimated version

3.7.0

### React Native version

0.72

### Platforms

iOS

### JavaScript runtime

None

### Workflow

None

### Architecture

None

### Build type

None

### Device

None

### Acknowledgements

Yes

**Comment by tomekzaw** [comment]
Should be fixed with #5372.

**Comment by Latropos** [comment]
The fix is merged: https://github.com/software-mansion/react-native-reanimated/pull/5372 🥳
